generated/niscope/session.py

```diff
@@ -2311,6 +2311,19 @@ def fetch(self, num_samples, timeout=5.0):
         '''
         return self._fetch(num_samples, timeout)
 
+    def fetch_array(self, num_samples, timeout=5.0):
+        import array
+
+        vi_ctype = visatype.ViSession(self._vi)  # case 1
+        channel_list_ctype = ctypes.create_string_buffer(self._repeated_capability.encode(self._encoding))  # case 2
+        timeout_ctype = visatype.ViReal64(timeout)  # case 9
+        num_samples_ctype = visatype.ViInt32(num_samples)  # case 9
+        wfm_array = array.array("d", [0]) * (num_samples * self._actual_num_wfms())
```

Comment:
minor: instead of 0, it should be 0.0, for correctness.
This may complicate generator enough that it's not worth it once we do for real.